ultimate_cron.module

```diff
@@ -217,292 +147,7 @@ function ultimate_cron_prepare_job($name, $hook, $job = NULL) {
- *
- * @todo: Port this.
- */
-function ultimate_cron_init() {
-  if (!variable_get('ultimate_cron_bypass_transactional_safe_connection')) {
-    $info = Database::getConnectionInfo('default');
-    Database::addConnectionInfo('default', 'ultimate_cron', $info['default']);
-  }
-  $GLOBALS['ultimate_cron_original_session_saving'] = drupal_save_session();
-  $GLOBALS['ultimate_cron_original_user'] = $GLOBALS['user'];
-
-  _ultimate_cron_variable_load('cron_last');
-}
-
-/**
- * Implements hook_help().
- *
- */
-function ultimate_cron_help($route_name, \Drupal\Core\Routing\RouteMatchInterface $route_match) {
-  switch ($route_name) {
-    case 'help.page.ultimate_cron':
-      // Return a line-break version of the module README.
-      return '<pre>' . file_get_contents(dirname(__FILE__) . '/README.txt') . '</pre>';
-  }
-}
-
-/**
- * Implements hook_cron_queue_info().
- *
- * Used for code injection in order to hijack cron runs.
- */
-function ultimate_cron_queue_info() {
-  $debug = debug_backtrace();
-  if (!empty($debug[3]['function']) && $debug['3']['function'] == 'drupal_cron_run') {
-    if (!empty($debug[4])) {
-      try {
-        if ($debug['4']['function'] == 'install_finished') {
-          // Looks like drupal_cron_run() was called unexpectedly.
-          \Drupal::logger('ultimate_cron')->info('Running cron in compatibility mode during site install.');
-        }
-        else {
-          // Looks like drupal_cron_run() was called unexpectedly.
-          \Drupal::logger('ultimate_cron')->warning('drupal_cron_run() called unexpectedly by @file:@line function @function. Running core in compatibility mode.', array(
-            '@function' => $debug['4']['function'],
-            '@line' => $debug['3']['line'],
-            '@file' => $debug['3']['file'],
-          ));
-        }
-
-        ultimate_cron_cron_run_compatibility();
-      }
-      catch (Exception $e) {
-        // Restore the user.
-        $GLOBALS['user'] = $GLOBALS['ultimate_cron_original_user'];
-        drupal_save_session($GLOBALS['ultimate_cron_original_session_saving']);
-        throw $e;
-      }
-      return array();
-    }
-
-    // Looks like drupal_cron_run() was called just the way we like it.
-    ultimate_cron_cron_run();
-    exit;
-  }
-  return array();
-}
-
-/**
- * Plugin clean up cron job.
- *
- * This is a cron callback for cleanup up plugins.
- *
- * @param CronJob $job
- *   The cron job being run.
- * @param array $arguments
- *   (optional) An array that can have one or more of the following elements:
- *   - type: The type of the plugin (settings, scheduler, launcher, logger).
- *   - name: The name of the plugin (queue, crontab, serial, database).
- */
-function ultimate_cron_plugin_cleanup($job, $arguments) {
-  $type = $arguments['type'];
-  $name = $arguments['name'];
-  $plugin = ultimate_cron_plugin_load($type, $name);
-  $plugin->cleanup();
-}
-
-/**
- * Implements hook_cronapi().
- *
- * Adds clean up jobs for plugins.
```

Comment:
Ah, here. We need to make sense of this in a separate task: "Add clean up job for plugins back".